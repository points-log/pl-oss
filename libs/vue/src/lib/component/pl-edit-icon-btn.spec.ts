import { shallowMount } from '@vue/test-utils';
import Vue from 'vue';
import Vuetify from 'vuetify';
import PLEditIconBtn from './pl-edit-icon-btn.vue';

Vue.use(Vuetify);

describe('PLEditIconBtn', () => {
  it('should match the snapshot', () => {
    const wrapper = shallowMount(PLEditIconBtn);
    expect(wrapper.element).toMatchSnapshot();
  });
});
