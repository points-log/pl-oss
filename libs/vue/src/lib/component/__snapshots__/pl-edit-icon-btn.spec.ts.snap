// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PLEditIconBtn should match the snapshot 1`] = `
<v-btn-stub
  activeclass=""
  icon="true"
  tag="button"
  type="button"
>
  <v-icon-stub>
    mdi-pencil
  </v-icon-stub>
</v-btn-stub>
`;
